process/sync/interface.go

```diff
@@ -0,0 +1,15 @@
+package sync
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+)
+
```

Comment:
add some comment, of what this interface is.

Reply:
Done